src/user-provisioning-service.js

```diff
@@ -0,0 +1,35 @@
+const dotenv = require('dotenv');
+const authApiProxy = require('./proxies/authenticateapi');
+const provision = require('./provision');
+
+//Used for the express HTTP listener
+const express = require('express');
+const bodyParser = require('body-parser');
+const {response} = require('express');
+const app = express();
+const port = 3000;
+
+dotenv.config();
+const filename = process.argv[2];
+
+//Main function.  Wrappering it with async to ensure that all async..await calls are properly fullfilled
+authApiProxy
```

Comment:
why not async/await?

Just reread: It actually says you are wrapping it in the comment above!

Reply:
Fixed. cut and paste error

Reply:
Part of that was a cut and paste error on the comments.